### PR TITLE
HostFeatures: Work around Qualcomm Oryon RNG errata

### PR DESCRIPTION
**Pull request by Sonicadvance1**
The Oryon is the first CPU we know of that implemented support for the RNG extension. It also has an errata where reading the RNDRRS register never returns success. X86's RDSEED guarantees forward progress with enough retries.

When an x86 processor messed this up at one point, some Linux systems would infinite loop (presumably when something in boot was filling an entropy pool). This required a microcode change to fix that processor.

The rdseed unittest infinite loops on this platform if RNG was exposed.

### CODE REVIEW

**Review by alyssarosenzweig — APPROVED** [review]
Ouch

File: FEXCore/Source/Interface/Core/HostFeatures.cpp [hunk 1]
```diff
@@ -44,6 +44,13 @@ static uint32_t GetFPCR() {
 static void SetFPCR(uint64_t Value) {
   __asm("msr FPCR, %[Value]" ::[Value] "r"(Value));
 }
+
+uint32_t GetMIDR() {
```
Comment (lioncash) [inline FEXCore/Source/Interface/Core/HostFeatures.cpp:48]: Seems like this should be static

**Review by lioncash — COMMENTED** [review]
One minor nit, but looks good overall

File: FEXCore/Source/Interface/Core/HostFeatures.cpp (hunk 1 above)
Comment (Sonicadvance1) [reply FEXCore/Source/Interface/Core/HostFeatures.cpp]: There we go.

**Review by lioncash — APPROVED** [review]

**Pull request merged**